v1/fleet_units/logrotate.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Log Rotator for docker containers
+After=docker.service
+Requires=docker.service
+
+[Service]
+User=core
+Restart=always
+ExecStartPre=-/usr/bin/docker stop logrotate
+ExecStartPre=-/usr/bin/docker rm -f logrotate
+ExecStartPre=/usr/bin/docker pull michaloo/logrotate
```

Comment:
you're still pulling michaloo.

might want to add something like:
`Environment=FD_IMAGE=behance/flight-director:latest`

then reference the variable in the pull/run commands